packages/protocol/contracts/governance/Governance.sol

```diff
@@ -859,93 +834,107 @@ contract Governance is
     return queue.contains(proposalId) && now < proposals[proposalId].timestamp.add(queueExpiry);
   }
 
-  // TODO(asa): Pass proposal as argument for gas optimization
   /**
-   * @notice Returns whether or not a particular proposal is passing according to the constitution.
+   * @notice Returns whether or not a particular proposal is passing according to the constitution
+   *   and the participation levels.
    * @param proposalId The ID of the proposal.
    * @return Whether or not the proposal is passing.
    */
-  function isProposalPassing(uint256 proposalId) public view returns (bool) {
-    Proposal storage proposal = proposals[proposalId];
+  function isProposalPassing(uint256 proposalId) external view returns (bool) {
+    return _isProposalPassing(proposals[proposalId]);
+  }
 
-    uint256 yesNoVotes = proposal.votes.yes.add(proposal.votes.no);
-    if (yesNoVotes == 0) {
-      return false;
-    }
-    int256 yesRatio = toFixed(proposal.votes.yes).divide(
-      toFixed(yesNoVotes)
+  function _isProposalPassing(Proposals.Proposal storage proposal) private view returns (bool) {
+    int256 support = proposal.adjustedSupport(
```

Comment:
I think it could be a bit tricky for someone to understand what is happening here from the function name. Can you think of any function names that would describe what "adjustment" needs to be made?

e.g: ``getParticipationAdjustedSupport`, `getSupportWithPaddedNoVotes`

Reply:
I'll try `getSupportWithQuorumPadding`